docs/design/roadmap.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Roadmap / Todos
+author: "@waalge"
+---
+
+Welcome the to CL Roadmap 🛣️ !
+
+This is a living document. Dates will be added as soon as funding is secured 😉
+(get in touch if you're interested).
+
+## Phase 0 :: Hub & Spoke
+
+This is our Concept phase. The aims are:
+
+- Become adept with Lightning design and concepts
+- Explore ways to apply Lightning to Cardano to greatest effect
+- Provide a POC/MVP of a Cardano Lightning network with a hub and spoke
+  topology.
+
+- M1 : Research and Specs
+  - [x] - Analysis of Bitcoin lightning
+  - [x] - L1 v0
+    - [ ] - Sub step
+  - [x] - L2 - Peer Protocol v0
+  - [ ] - L2 - Hub v0
+- M2 : Validator(s)
+  - [ ] - Validator v0 implementation
+  - [ ] - Benchmarking and tests
+  - [ ] - Validator v1, with sub step)
+- M3 : L2 Stack and Key Management
+  - [ ] - CI/CD
+  - [ ] - Networking layer
+  - [ ] - Persistence layer
+  - [ ] - Signing server and proxy key setup
+- M4 : Tx builders
+  - [ ] - Tx building
+  - [ ] - L1 abstraction/ interface
+  - [ ] - L1 provider implementation
+- M5 : Cli
+  - [ ] - Channel management w/ cli tool
```

Comment:
```suggestion
  - [ ] - Channel management cli tool
```
?
This can be also a tiny shell which preserves some state etc.

Reply:
Good question. Maybe, or requires a long running service?